codebuild/integ_codebuild.config

```diff
@@ -0,0 +1,81 @@
+# Config file consumed by create_project
+# Helpful reminder about CodeBuild provided docker images:
+#  https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html
+[Global]
+stack_name: s2nIntegrationScheduled
+
+#Reusable templates - use the snippet:NAME
+[IntegUbuntuBoilerplateLarge]
+image : aws/codebuild/standard:2.0
+env_type: LINUX_CONTAINER
+compute_type: BUILD_GENERAL1_LARGE
+timeout_in_min: 90
+buildspec: codebuild/spec/buildspec_ubuntu_integ_openssl102.yml
+source_location: https://github.com/awslabs/s2n.git
+source_type : GITHUB
+source_clonedepth: 1
+source_version:
+
+# Boring + GCC9, Libre + GCC6
+[CodeBuild:s2nIntegrationBoringLibre]
+image : aws/codebuild/standard:2.0
+env_type: LINUX_CONTAINER
+compute_type: BUILD_GENERAL1_LARGE
+timeout_in_min: 90
+buildspec: codebuild/spec/buildspec_ubuntu_integ_boringlibre.yml
+source_location: https://github.com/awslabs/s2n.git
+source_type : GITHUB
+source_clonedepth: 1
+source_version:
+env: TESTS=integration BUILD_S2N=true
+
+# OpenSSL111 + GCC6 + Corked and notCorked + Gcc4.8
```

Comment:
Is this the only yaml file providing corked io? Any reason not to add a corked io options to the rest of the integ runs as well?

Reply:
This just duplicates what is in travis now, it could be expanded if runtimes are too long.